sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodShouldNotOnlyReturnConstant.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Tests.Diagnostics
+{
+    public class Program
+    {
+        public int GetInt() // Noncompliant {{Remove this method and declare a constant for this value.}}
+//                 ^^^^^^
+        {
+            return 12;
```

Comment:
Does it raise when you do `return 3 * 2;`?

Reply:
It doesn't raise.